tests/sav/base_gen_redef_alt3.res

```diff
@@ -1 +1 @@
-alt/base_gen_redef_alt3.nit:17,13: Redef error: Formal parameter arity missmatch; got 2, expected 1.
+alt/base_gen_redef_alt3.nit:17,13: Redef Error: number of formal parameters mismatch; got 2, expected 1.
```

Comment:
I would be worth inverting this message too: `Redef Error: got 2 formal parameters, expected 1.`